ngraph/frontend/frontend_manager/include/frontend_manager/frontend_manager.hpp

```diff
@@ -119,4 +95,31 @@ namespace ngraph
 
     } // namespace frontend
 
+    template <>
+    class FRONTEND_API VariantWrapper<std::shared_ptr<std::istream>>
+        : public VariantImpl<std::shared_ptr<std::istream>>
+    {
+    public:
+        static constexpr VariantTypeInfo type_info{"Variant::std::shared_ptr<std::istream>", 0};
+        const VariantTypeInfo& get_type_info() const override { return type_info; }
+        VariantWrapper(const value_type& value)
+            : VariantImpl<value_type>(value)
+        {
+        }
+    };
```

Comment:
This is complete template specialization. Why do we need to define it in a header file? It is explicitly used internally only.

Reply:
not working if move to cpp